editor/src/components/canvas/canvas-strategies/strategies/flex-reorder-strategy.spec.browser2.tsx

```diff
@@ -434,7 +663,6 @@ describe('Flex Reorder Strategy', () => {
 
       const startPoint = windowPoint({ x: targetElementBounds.x + 5, y: targetElementBounds.y + 5 })
       await mouseClickAtPoint(canvasControlsLayer, startPoint, { modifiers: cmdModifier })
-      await pressKey('Escape')
```

Comment:
why this change?

Reply:
as far as I could see, if I press `esc` after selecting an element, the parent is selected, and I wanted to select the actual element that the test clicks on